### ISSUE TITLE
Black Screen After Switching To Another App/Home Screen

### ISSUE THREAD

**Issue by MotorBottle** [utmapp/UTM#6018]
**Describe the issue**
A clear and concise description of what the issue is. **BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

**Configuration (required)**
* UTM Version: 
* OS Version: 
* Device Model: 
* Is it jailbroken (name jailbreak used)? 
* How did you install UTM? 

**Crash log**
If the app crashed, you need a crash log. To get your crash log, open the Settings app and browse to `Privacy -> Analytics & Improvements -> Analytics Data` and find the latest entry for UTM. You should export the text and attach it here.

**Debug log**
For all issues, _including_ crashes, you should attach a debug log. Open UTM, and open the settings for the VM you wish to launch. Near the bottom of the configuration options (or at the top of the `QEMU` page) is `Debug Log`. Turn it on and save the VM. After you experience the issue, quit the VM and re-launch UTM. Open the VM settings again and select `Export Log...` and attach it here.

**Upload VM**
(Optional) If possible, upload the `config.plist` inside the `.utm`. If you do not have this, you can upload the entire `.utm` but note this contains your personal data. Since GitHub has an attachment size limit, you may want to upload to another service such as Google Drive. Link it here.

**Comment by imkion** [comment]
I'm in the same situation as you, but I have founded a solution, after entering the system in UTM app , tap once with your finger or 3 times in a row on the 3 dots at the top of the screen, then select Slide Side, then open another app, and finally minimize the UTM app, after doing so, the next time you open the UTM app, the screen will not go black.

**Comment by imkion** [comment]
> ![image](https://private-user-images.githubusercontent.com/50960678/326193249-ce3cdfec-714d-46a0-82c1-9eaf2eaf374a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQyMzY5NzksIm5iZiI6MTcxNDIzNjY3OSwicGF0aCI6Ii81MDk2MDY3OC8zMjYxOTMyNDktY2UzY2RmZWMtNzE0ZC00NmEwLTgyYzEtOWVhZjJlYWYzNzRhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI3VDE2NTExOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE0OTFmOTc4NmNhNDNjOGQ2MTM2ZjFmYjVmMzVkYzg0ODYxZDg3MGE5NzgwOTA4YjEwYzRjMTMyOTkwODlmZjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.SPHmRfFYT821NtmaX7BsQCA2yWvSMDcdy4uhrLLGoB0)
> 
> curious if the run in background option is selected

Yes, I have selected the run in background option, but the screen is still black after Switching To Another App/Home Screen. However, I solved it temporarily through the above method. My other iOS 15.1 iPad Pro 2021 does not work like this.

**Comment by imkion** [comment]
It seems that this bug is 100% reproducible on iOS 16.3 devices.

**Comment by MotorBottle** [comment]
An update on my case guys. I updated UTM to 4.5.0beta and 4.5.1beta. The 2 versions partially solved the problem. When I retrun from home/other app, the view would still turn black but moving the cursor could respawn the view (But the wallpaper did not show up directly, only other elements of the UI. Wallpaper could show up if I rotated the screen or wait for a while, but this could not be precise cuz I'm not using it lately).

**Comment by imkion** [comment]
> An update on my case guys. I updated UTM to 4.5.0beta and 4.5.1beta. The 2 versions partially solved the problem. When I retrun from home/other app, the view would still turn black but moving the cursor could respawn the view (But the wallpaper did not show up directly, only other elements of the UI. Wallpaper could show up if I rotated the screen or wait for a while, but this could not be precise cuz I'm not using it lately).

You can try the method I mentioned earlier,You can use split screen browsing to minimize UTM app and then switch to other app/home screen, which can solve the problem perfectly.

**Comment by osy** [comment]
Another thing to try: after getting the black screen, try switching to split screen or rotating the iPad to cause a screen resolution change. I'm wondering if there's some Metal framebuffer that's getting invalidated.

Does this happen if you use a graphics card that doesn't support GPU acceleration?

**Comment by MotorBottle** [comment]
I did test on various of settings on 4.5.1beta. Here are the results.

1st thing: Render Backend switching showes both Angel OGL and Angel Metal have this issue.

2nd thing: Rotation of screen did helped making the view getting normal.

3rd thing: A new problem I found during the test which did not happen in 4.4.5. After swtiching back from home screen, the rotation sometimes appear to be inappropriate, like when ipad and vm view was horizontal, and I switch to home then wait 5 seconds and back, it would turn vertical. But could be turned normal by rotating the ipad for couple of times.

4th thing: Your hypothesis was correct, choosing a non-acceleration supported gpu would not perform the black screen issue, but the rotation issue described in 3rd also happens.

**Comment by MotorBottle** [comment]
> I'm in the same situation as you, but I have founded a solution, after entering the system in UTM app , tap once with your finger or 3 times in a row on the 3 dots at the top of the screen, then select Split screen browsing, then select to open another app, and drag the dividing line in the middle to finally minimize the UTM app, after doing so, the next time you open the UTM app, the screen will not go black.

Well, I tried as you said but still got the rotation and black screen issue happened
